pkg/chunked/cache_linux_test.go

```diff
@@ -211,3 +215,60 @@ func TestUnmarshalToc(t *testing.T) {
 	_, err = unmarshalToc([]byte(jsonTOC + "123"))
 	assert.Error(t, err)
 }
+
+func benchmarkLookup(b *testing.B, sizeCache, n int) {
```

Comment:
I think I'll drop these tests (and the `chunked: add benchmark tests for lookup` commit) from the series since these tests are already covered in `bloom_filter_test.go`.

With the tests in `bloom_filter_test.go`, I've:

```
$ benchstat <(go test -bench=. -count 20 -benchtime=1x)
name                  time/op
LookupBloomFilter-12  8.24ms ±10%
LookupBloomRaw-12      196ms ± 7%
```